apps/style/code-studio/petition.scss

```diff
@@ -1,12 +1,66 @@
 @import 'font';
 
 .petition-block {
-  width: 970px;
```

Comment:
This was used in storybook, but container width is set application-wide on the site